backend/backend.go

```diff
@@ -371,31 +368,12 @@ func (backend *Backend) Coin(code coinpkg.Code) (coinpkg.Coin, error) {
 			"https://etherscan.io/tx/",
 			etherScan,
 			nil)
-	case code == coinpkg.CodeRETH:
-		etherScan := etherscan.NewEtherScan("https://api-rinkeby.etherscan.io/api", backend.etherScanHTTPClient)
-		coin = eth.NewCoin(etherScan, code, "Ethereum Rinkeby", "RETH", "RETH", params.RinkebyChainConfig,
-			"https://rinkeby.etherscan.io/tx/",
-			etherScan,
-			nil)
-	case code == coinpkg.CodeTETH:
-		etherScan := etherscan.NewEtherScan("https://api-ropsten.etherscan.io/api", backend.etherScanHTTPClient)
-		coin = eth.NewCoin(etherScan, code, "Ethereum Ropsten", "TETH", "TETH", params.RopstenChainConfig,
-			"https://ropsten.etherscan.io/tx/",
-			etherScan,
-			nil)
 	case code == coinpkg.CodeGOETH:
 		etherScan := etherscan.NewEtherScan("https://api-goerli.etherscan.io/api", backend.etherScanHTTPClient)
 		coin = eth.NewCoin(etherScan, code, "Ethereum Goerli", "GOETH", "GOETH", params.GoerliChainConfig,
 			"https://goerli.etherscan.io/tx/",
 			etherScan,
 			nil)
-	case code == coinpkg.CodeERC20TEST:
-		etherScan := etherscan.NewEtherScan("https://api-ropsten.etherscan.io/api", backend.etherScanHTTPClient)
-		coin = eth.NewCoin(etherScan, code, "ERC20 TEST", "TEST", "TETH", params.RopstenChainConfig,
-			"https://ropsten.etherscan.io/tx/",
-			etherScan,
-			erc20.NewToken("0x2f45b6fb2f28a73f110400386da31044b2e953d4", 18),
-		)
```

Comment:
Was this for some kind of test?

Not needed anymore?

Reply:
I once deployed (I think? can't quite remember) a ERC20 token for manual testing. Haven't used it in years.